runtime.go

```diff
@@ -1614,6 +1614,11 @@ func (r *Runtime) ToValue(i interface{}) Value {
 		return intToValue(int64(i))
 	case int64:
 		return int64ToValue(i)
+	case time.Time:
+		// Preserve the time in case it is unmodified
+		obj := r.newDateObject(i, true, r.global.DatePrototype)
+		obj.__wrapped = i
```

Comment:
🤔 I might have confused myself... Why do we need to add the time to `__wrapped`? Is there an issue with how it's converted to `mscs`? In other words, what would happen if we just did `return r.newDateObject(i, true, r.global.DatePrototype)` in here?

Reply:
there's an issue with how it's created to mscs. I suspect this is why the owners of the repo opted for time.Time to be a goval instead of converting to date.